src/components/RegistrationList.tsx

```diff
@@ -16,10 +16,12 @@ interface RegistrationListProps {
 }
 
 export const RegistrationList = ({ registrations }: RegistrationListProps) => (
-  <List>
+  <List disablePadding>
     {registrations.map((registration) => (
       <ErrorBoundary key={registration.id}>
-        <RegistrationListItem registration={registration} />
+        <SearchListItem sx={{ borderLeftColor: 'registration.main' }}>
+          <RegistrationListItem registration={registration} />
```

Comment:
2 stk `ListItem`? nejtack
```suggestion
        <SearchListItem sx={{ borderLeftColor: 'registration.main' }}>
          <RegistrationListItemContent registration={registration} />
```